frontend/src/queries/nodes/DataVisualization/Components/Table.tsx

```diff
@@ -102,7 +104,7 @@ export const Table = (props: TableProps): JSX.Element => {
             dataSource={tabularData}
             columns={tableColumns}
             loading={responseLoading}
-            pagination={{ pageSize: 2000 }}
+            pagination={{ pageSize: DEFAULT_PAGE_SIZE }}
```

Comment:
Thought you just increased this to 2000?

Reply:
oops